crates/key-value/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use key_value::*;
+use serde_json;
+use std::collections::HashMap;
+use std::io::Read;
+
+pub use key_value::add_to_linker;
+
+wit_bindgen_wasmtime::export!("../../wit/ephemeral/key-value.wit");
+
+#[derive(Default, Clone)]
+pub struct KeyValue {
+    pub(crate) db: HashMap<String, Vec<u8>>,
+}
+
+impl KeyValue {
+    pub fn load_json() -> Result<KeyValue, anyhow::Error> {
+        // TODO: Config this path
+        let mut file = match std::fs::File::open("db.json") {
```

Comment:
Spin stores logs in `~/.spin/<appid>/logs/<componentid_<stderr>/<stdout>.txt`.

Would it make sense to store the JSON file by default in something similar to `~/.spin/appid/key-value/<>.json`, potentially with the possibility to override that when starting Spin?